worlds/terraria/Options.py

```diff
@@ -0,0 +1,39 @@
+from Options import Choice, Option
+import typing
+
+class Goal(Choice):
+    """The victory condition for your run. Stuff after the goal will not be shuffled."""
+    display_name = "Goal"
+    option_wall_of_flesh = 0
+    option_plantera = 1
+    option_moon_lord = 2
+    option_zenith = 3
+    default = 2
+
+class Achievements(Choice):
+    """
+    Adds checks upon collecting achievements. Achievements for clearing bosses and events are excluded.
+    "Exclude Grindy" also excludes fishing achievements.
+    """
+    display_name = "Achievements"
+    option_none = 0
+    option_exclude_grindy = 1
+    option_exclude_fishing = 2
+    option_all = 3
+    default = 1
+
+class FillExtraChecksWith(Choice):
+    """
+    Applies if you have achievements enabled. "Useful Items" helps to make the early game less grindy.
+    Items are rewarded to all players in your Terraria world.
+    """
+    display_name = "Fill Extra Checks With"
+    option_nothing = 0
```

Comment:
I'm not much a fan of the option to put lots of `Nothing`s into the pool. There's just something a little extra feelsbadman about doing a location check and finding literally nothing (compared to finding a junk item). I'd personally just have it always add the items.

Reply:
Some players might feel like the item rewards aren't very vanilla or speed up the game too much for longer runs. It's the player's option, and item rewards are the default anyway, so I feel like it can't hurt.

Reply:
Maybe you could have an option to limit the items to less useful ones like informational accessories (stopwatch, tally counter, etc), paint sprayer, etc

Reply:
guess I'm not sure there's enough of those to fill out the pool. Could always add gold as a filler item?

Reply:
I doubt that option would see much use. I would accept a change for it, though. Can't hurt

Reply:
Alright, multiple people are asking for it, so I'll add it 👍